src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -4316,6 +4316,19 @@ paths:
       parameters:
         - $ref: '#/parameters/personIdParam'
         - $ref: '#/parameters/fieldsParam'
+        - name: client
+          in: query
+          description: |
+            Optionally provided in the request query to customise the sent email.
+            **Note:** The client must be registered before this API can send an email.
+          type: string
+          required: false
+        - name: workspacePath
+          in: query
+          description: |
+            Optionally override the workspace deployment links to customise the sent email.
```

Comment:
I don't think this is descriptive enough, it's not clear to me what effect this parameter has. If it's a URL we'll need to be extra careful we don't open security issues.